packages/backend/discovery/arbitrum/config.jsonc

```diff
@@ -105,6 +106,16 @@
         "l2Dai"
       ]
     },
+    "ValidatorUtils": {
+      "fields": {
+        "ARBITRUM_STAKERS": {
+          "type": "call",
+          "method": "function getStakers(address rollup, uint64 startIndex, uint64 max) view returns (address[], bool hasMore)",
+          "args": ["0x5eF0D09d1E6204141B4d37530808eD19f60FBa35", 0, 100]
```

Comment:
Can't we get this address dynamically? I remember we had `{{ value_name}}` notation in those configs.

Reply:
`Call a method and reference other fields in arguments:` unfortunately this contract does not have this address stored